src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const currentObject = { ...state };
+  const clonesArray = [];
+
+  for (const i in actions) {
+    switch (actions[i].type) {
```

Comment:
```suggestion
    switch (action.type) {
```
You can also use desctructuring to get all the properties of action

Reply:
yep. that looks better